### PR TITLE
grub as a new chef resource

### PR DESCRIPTION
**Pull request by rmcleod8**
existing libraries resource was causing issues with latest chefspec

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,6 @@
...
-## [Unreleased]
-
-## [1.0.0]
-### Bug Fixes
-- Fixes integration test harness using Test Kitchen.
-- Squashes several issues while testing locally and on machines.
-
-[Unreleased]: https://bbgithub.dev.bloomberg.com/chef/blp-base-cookbook/compare/v1.0.0...HEAD
-[1.0.0]: https://bbgithub.dev.bloomberg.com/chef/blp-base-cookbook/compare/v1.0.0...HEAD
+## [0.2.0]
```
Comment (jjustice6) [inline CHANGELOG.md:5]: This should definitely be a first-digit bump if we go with this.

File: resources/grub.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  end
+  grub_config
+end
+
+action :create do
+  notifying_block do
+    unless new_resource.stringify.empty?
+      file new_resource.path do
+        path new_resource.path
+        content new_resource.stringify
```
Comment (jjustice6) [inline resources/grub.rb:27]: You don't need to do `new_resource.` when calling a function; that's only for getting properties.

File: resources/grub.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#  Resource to manage the Grub configuration file
+#
+provides :grub
+
+property :path, String, name_property: true
+property :settings, Hash, default: {}
+
+def stringify
+  grub_config = ''
+  s = settings
```
Comment (jjustice6) [inline resources/grub.rb:10]: I don't like this and I suspect it'll throw errors/warnings. Should be `new_resource.settings`; or make it a parameter to `stringify`.

**Comment by rmcleod8** [comment]
Going to blp-grub, closing.

**Pull request closed without merging**